modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/service/test/JournalFolderServiceTest.java

```diff
@@ -787,10 +851,23 @@ public void testUpdateParentWithRestriction() throws Exception {
 		Assert.assertEquals("Description 1", parentFolder.getDescription());
 	}
 
+	protected JournalFolder addJournalFolder(
+			String externalReferenceCode, String name, String description)
+		throws Exception {
+
+		return _journalFolderFixture.addFolder(
+			externalReferenceCode,
+			JournalFolderConstants.DEFAULT_PARENT_FOLDER_ID, name, description,
+			ServiceContextTestUtil.getServiceContext(
+				_group, TestPropsValues.getUserId()));
+	}
+
 	private static final Log _log = LogFactoryUtil.getLog(
 		JournalFolderServiceTest.class);
 
 	@DeleteAfterTestRun
 	private Group _group;
 
+	private JournalFolderFixture _journalFolderFixture;
```

Comment:
As we create this object in the `setUp` method, we should ensure that it will be deleted after Run the tests (like it is doing with `_group`)
So, just put the annotation `@DeleteAfterTestRun`

Reply:
@LuismiBarcos I have tried using `@DelteAfterTestRun` and it seems that it doesn't work well because it seems that once the test have been executed, there is no entity in the DB to be deleted.

Reply:
The idea that @ccorreagg has come with and which makes sense and works well is to inject the `JournalFolderLocalService` in `JournalFolderFixture`. This avoid the tests to use a not Null service in `JournalFolderFixture`